lib/hawkular/inventory/entities.rb

```diff
@@ -119,6 +119,7 @@ class OperationDefinition < BaseEntity
     def initialize(op_hash)
       super(op_hash)
       @params = {}
+      return if op_hash['properties'].nil?
```

Comment:
maybe worth to check also `.empty?` ?  in case it's not nil but is an empty hash for some reason.